apps/fares/lib/format.ex

```diff
@@ -28,6 +28,7 @@ defmodule Fares.Format do
   def media(:cash), do: "Cash"
   def media(:senior_card), do: "Senior CharlieCard or TAP ID"
   def media(:student_card), do: "Student CharlieCard"
+  def media(:paper_ferry), do: "Paper Ferry Ticket"
```

Comment:
Looking at the ticket, I believe this should not be title case, but lowercase, unless you and the content team already discussed this.

Reply:
Yes, content decided to do it uppercase for consistency